dev/breeze/src/airflow_breeze/commands/setup_commands.py

```diff
@@ -269,6 +269,34 @@ def dict_hash(dictionary: dict[str, Any]) -> str:
     return dhash.hexdigest()
 
 
+def is_common_param(name):
+    return name == "verbose" or name == "help" or name == "dry_run"
+
+
+def is_short_flag(opt):
+    return len(opt) == 2 and (not opt.startswith("--"))
+
+
+def validate_params_for_command(command_params, command):
+    options_command_map = {}
+    if "params" in command_params:
+        for param in command_params["params"]:
+            name = param["name"]
+            if not is_common_param(name):
```

Comment:
I think this whole part abot checking common params can be removed now.

Reply:
Right it is not valid anymore